src/main/java/it/pagopa/pm/gateway/service/ClientsConfigService.java

```diff
@@ -0,0 +1,35 @@
+package it.pagopa.pm.gateway.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.pagopa.pm.gateway.dto.ClientConfig;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@Scope("singleton")
+public class ClientsConfigService {
+    @Value("${pgs.clients.config}")
```

Comment:
Please update `.env.local` with the new configuration variable

Reply:
https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/196/commits/9a8a7becba2a16f4762c739cd3fd79cfc9f03c4b